### ISSUE TITLE
PLEASE READ: When opening a bug report, please give us actual good info about your problem...

### ISSUE THREAD

**Issue by EnderBro123** [EnderBro123/Modpacks#4]
Hi!

If you are reading this, then that might mean your awful at giving us a good bug report... Well, time to teach you something. If your getting support from me or Mario, please for the love of god give the info we ask for. If your 1 or 2 versions behind on a project of ours, then update. Simple as that and to the point. No support is given to old versions. We'll simply ignore and instantly close the issue if your having issues with that outdated version your on. Also, if a mod is causing a crash of some sort, then please just report it to the respective mod developer. Not us. We have nothing to do with the inner code of the mods we use within our modpacks. We'll push updates to our packs when needed. It may even just be your PC, Java, OpenGL or something else unrelated to Minecraft not being fully updated either.

In a nutshell, just follow the instructions we give you, and your good to go. Don't say "I'm using the latest version" when giving us info. That's not helpful AT ALL. Simply give us the necessary info, and we're all good.
